code/stable/swhsnopcm/SRS/mdBook/src/SecFRs.md

```diff
@@ -34,6 +34,7 @@ Output-Values: Output \\({T\_{\text{W}}}\\)(\\(t\\)) (from [IM:eBalanceOnWtr](./
 |\\({A\_{\text{tol}}}\\)  |Absolute tolerance                                         |--                                                   |
 |\\({C\_{\text{W}}}\\)    |Specific heat capacity of water                            |\\(\frac{\text{J}}{\text{kg}{}^{\circ}\text{C}}\\)   |
 |\\(D\\)                  |Diameter of tank                                           |\\({\text{m}}\\)                                     |
+|\\({E\_{\text{W}}}\\)    |Change in heat energy in the water                         |\\({\text{J}}\\)                                     |
```

Comment:
I guess this is one of the good things about this change? The outputs are gathered from `SystemInformation`'s designated outputs, so it looks like it found one more to add!